flixel/input/gamepad/OUYAButtonID.hx

```diff
@@ -16,8 +16,8 @@ class OUYAButtonID
 	public static inline var A:Int = 1;
 	public static inline var LB:Int = 6;
 	public static inline var RB:Int = 7;
-	public static inline var LEFT_ANALOG:Int = 10;
-	public static inline var RIGHT_ANALOG:Int = 11;
+	public static inline var LEFT_STICK_BTN:Int = 10;
```

Comment:
Can this be resolved by just expanding to "BUTTON" or would you prefer if it said "CLICK" ? The purpose being simply to disambiguate the analog from the digital input that occupy the same physical control.